app.js

```diff
@@ -1,16 +1,26 @@
 const express = require("express");
 const { getCategories } = require("./controllers/categories.controllers");
+const { getReviewById } = require("./controllers/reviews.controllers");
 
 const app = express();
 
 app.get("/api/categories", getCategories);
 
+app.get("/api/reviews/:review_id", getReviewById);
+
 app.get("*", (req, res) => {
-    res.status(404).send({msg: "Not found"})
-})
+  res.status(404).send({ msg: "Not found" });
+});
 
-app.use((err,req,res) => {
-    res.status(500).send({msg: "Internal Server Error"})
-})
+app.use((err, req, res, next) => {
```

Comment:
You may want to look at shortly splitting these out into another file as error handlers ( should be split by functionality (eg handleCustomErrors, handlePsqlErrors)) then requiring them in to app.js :)